horace_core/lattice_functions/refine_crystal.m

```diff
@@ -167,9 +167,12 @@
 end
 
 kk = multifit (vcryst0_3, zeros(3*nv,1), 0.01*ones(3*nv,1));
+% function to caclulate distance between actual and expected bragg peak
+% positions in reciprocal space.
 kk = kk.set_fun (@reciprocal_space_deviation, {pars,rlu_expected}, pfree, pbind);
 kk = kk.set_options ('list', 0);
 kk = kk.set_options ('fit',[1e-4,50,-1e-6]);
+clOb = set_temporary_config_options('hpc_config','parallel_multifit',false);
```

Comment:
this appears to be the only part of this PR that actually makes a change; the rest is refactoring and testing. There is no comment here to say - parallel multifit disabled here due to ticket 1724. The reader should be aware why this line is here and be able to trace it through the system. Please add the comment

Reply:
`set_temporary_config_options` is only available during testing (it's in `test/common_functions`. It cannot be used outside.

Reply:
Or you have moved it in this PR without saying that anywhere

Reply:
I've moved it into `herbert/utilities`